spec/deck_spec.rb

```diff
@@ -0,0 +1,79 @@
+    card_3 = Card.new("Describe in words the exact direction that is 697.5° clockwise from due north?", "North north west", :STEM)
+
+    cards = [card_1, card_2, card_3]
+
+    deck = Deck.new(cards)
+
+    expect(deck).to be_instance_of(Deck)
+  end
+
+  it 'has cards' do
+    card_1 = Card.new("What is the capital of Alaska?", "Juneau", :Geography)
+    card_2 = Card.new("The Viking spacecraft sent back to Earth photographs and reports about the surface of which planet?", "Mars", :STEM)
+    card_3 = Card.new("Describe in words the exact direction that is 697.5° clockwise from due north?", "North north west", :STEM)
+
+    cards = [card_1, card_2, card_3]
+
+    deck = Deck.new(cards)
+
+    expect(deck.cards).to eq([card_1, card_2, card_3])
+  end
+
+  it 'has card count' do
+    card_1 = Card.new("What is the capital of Alaska?", "Juneau", :Geography)
+    card_2 = Card.new("The Viking spacecraft sent back to Earth photographs and reports about the surface of which planet?", "Mars", :STEM)
+    card_3 = Card.new("Describe in words the exact direction that is 697.5° clockwise from due north?", "North north west", :STEM)
+
+    cards = [card_1, card_2, card_3]
+
+    deck = Deck.new(cards)
+
+    expect(deck.count).to eq(3)
+  end
+
+  it 'has STEM' do
+    card_1 = Card.new("What is the capital of Alaska?", "Juneau", :Geography)
+    card_2 = Card.new("The Viking spacecraft sent back to Earth photographs and reports about the surface of which planet?", "Mars", :STEM)
+    card_3 = Card.new("Describe in words the exact direction that is 697.5° clockwise from due north?", "North north west", :STEM)
+
+    cards = [card_1, card_2, card_3]
+
+    deck = Deck.new(cards)
+
+    expect(deck.cards_in_category(:STEM)).to include(card_2)
+    expect(deck.cards_in_category(:STEM)).to include(card_3)
+  end
+
+  it 'has Geography' do
+    card_1 = Card.new("What is the capital of Alaska?", "Juneau", :Geography)
+    card_2 = Card.new("The Viking spacecraft sent back to Earth photographs and reports about the surface of which planet?", "Mars", :STEM)
+    card_3 = Card.new("Describe in words the exact direction that is 697.5° clockwise from due north?", "North north west", :STEM)
+
+    cards = [card_1, card_2, card_3]
+
+    deck = Deck.new(cards)
+
+    expect(deck.cards_in_category(:Geography)).to include(card_1)
+  end
+
+  it 'has Pop Culture' do
+    card_1 = Card.new("What is the capital of Alaska?", "Juneau", :Geography)
+    card_2 = Card.new("The Viking spacecraft sent back to Earth photographs and reports about the surface of which planet?", "Mars", :STEM)
+    card_3 = Card.new("Describe in words the exact direction that is 697.5° clockwise from due north?", "North north west", :STEM)
+
+    cards = [card_1, card_2, card_3]
+
+    deck = Deck.new(cards)
+
+    expect(deck.cards_in_category(:Pop_culture)).to include() #probably wrong, need to test more. Maybe !cards, or nil?
```

Comment:
Your test here is incomplete. Make sure to copy and paste directly from the interaction pattern.